app/views/events/_show_event.html.erb

```diff
@@ -20,6 +20,16 @@
   <dd><%= @event.knowledge_level %></dd>
   <dt><strong><%= model_class.human_attribute_name(:application_deadline) %>:</strong></dt>
   <dd> <%= I18n.l(@event.application_deadline) + ' ' + I18n.t('midnight') %></dd>
+  <% if can? :manage, Event %>
```

Comment:
`:manage` is the broadest permission, maybe `:create` would be more appropriate. Currently, it wouldn't make much of a difference, though.